src/test/regress/sql/failure_savepoints.sql

```diff
@@ -0,0 +1,190 @@
+SELECT citus.mitmproxy('conn.allow()');
+
+SET citus.shard_count = 2;
+SET citus.shard_replication_factor = 1; -- one shard per worker
+SET citus.next_shard_id TO 100950;
+ALTER SEQUENCE pg_catalog.pg_dist_placement_placementid_seq RESTART 150;
+
+CREATE TABLE artists (
+    id bigint NOT NULL,
+    name text NOT NULL
+);
+SELECT create_distributed_table('artists', 'id');
+
+-- add some data
+INSERT INTO artists VALUES (1, 'Pablo Picasso');
+INSERT INTO artists VALUES (2, 'Vincent van Gogh');
+INSERT INTO artists VALUES (3, 'Claude Monet');
+INSERT INTO artists VALUES (4, 'William Kurelek');
+
+SELECT citus.clear_network_traffic();
```

Comment:
nit; I don't think this is required for the tests if you don't list the recorded network traffic